src/js/easymde.js

```diff
@@ -1945,7 +1945,12 @@ EasyMDE.prototype.autosave = function () {
             this.options.autosave.loaded = true;
         }
 
-        localStorage.setItem('smde_' + this.options.autosave.uniqueId, easyMDE.value());
+        var value = easyMDE.value();
+        if (value != '') {
```

Comment:
```suggestion
        if (value !== '') {
```

It's better to always use tripe-equals

Reply:
Fixed!